src/tools/wasm2c.cc

```diff
@@ -94,6 +94,7 @@ static void ParseOptions(int argc, char** argv) {
     exit(1);
   }
   s_features.disable_bulk_memory();
+  s_features.enable_multi_memory();
```

Comment:
I'm not sure we want to enable this in add cases.  The tests that need it can/should probably enable it on the command line  (see  test/spec/multi-memory/data.txt for example)